sdk/keyvault/keyvault-keys/README.md

```diff
@@ -248,9 +248,22 @@ await client.recoverDeletedKey(keyName);
 // await client.purgeDeletedKey(keyName);
 ```
 
-Since the deletion of a key won't happen instantly, some time is needed
-after the `deleteKey` method is called before the deleted key is
-available to be read, recovered or purged.
+Since Keys take some time to get fully deleted, `beginDeleteKey`
+returns a Poller object that keeps track of the underlying Long Running
+Operation according to our guidelines:
+https://azure.github.io/azure-sdk/typescript_design.html#ts-lro
+
+Once you receive the poller, you'll be able to either run individual service
+calls until the key is deleted, or wait until the process is done:
+
+```typescript
+const poller = await client.beginDeleteKey(certificateName, certificatePolicy);
+
+await poller.poll(); // One service call
+
+const deletedKey = await poller.pollUntilDone();
```

Comment:
I would add a comment above this like "alternatively, keep polling automatically until the operation finishes."